.github/dependabot.yml

```diff
@@ -22,3 +25,92 @@ updates:
   reviewers:
   - projectcontour/maintainers
   
+# release branch N targets
+- target-branch: release-1.25
+  package-ecosystem: "gomod"
+  directory: "/"
+  schedule:
+    interval: "weekly"
+  ignore:
+  - update-types:
+    - "version-update:semver-major"
+    - "version-update:semver-minor"
```

Comment:
docs aren't totally clear, but do we need need to add a `dependency-name: *` here? ref. https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore

Reply:
probably won't hurt to do so, let's try it!